server/game/cards/03-WC/DeepwoodDruid.js

```diff
@@ -0,0 +1,21 @@
+const Card = require('../../Card.js');
+
+class DeepwoodDruid extends Card {
+    setupCardAbilities(ability) {
+        this.play({
+            reap: true,
+            target: {
+                cardType: 'creature',
+                optional: false,
```

Comment:
Optional: false is the default so this is redunant

Reply:
fixed in latest pr.